tests/wpt/mozilla/tests/css/table_rowspan_rowgroup_a.html

```diff
@@ -0,0 +1,23 @@
+<!DOCTYPE html>
+<html>
+  <meta charset="UTF-8">
+  <link rel="match" href="table_rowspan_rowgroup_ref.html">
```

Comment:
I thought spec links used to be `<link rel=help>`, but I don't think it matters much anyway :)